package.json

```diff
@@ -69,6 +69,7 @@
     "axe-webdriverjs": "0.4.0"
   },
   "dependencies": {
+    "@okta/courage": "1.14.0",
```

Comment:
yeah, this should not be here.
There is already an entry for courage in ```optionalDependencies``` and that is the one we need to update